app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,9 @@
         <div class="menu-item">
           <%= menu_link_to 'All users', users_path %>
         </div>
+        <div class="menu-item">
+          <%= menu_link_to 'Profile', current_user %>
+        </div>
```

Comment:
You should add a menu to check for the received and sent invitations. Currently, you have that on the All Users menu, but it's not the best place to put it (I didn't find them easily). You can call it Notifications.

**Optional** You can make use of the friendship model and add an icon to show the number of friendship requests a user has, next to the menu title.